Resources/Prototypes/Entities/Clothing/Back/smuggler.yml

```diff
@@ -0,0 +1,1009 @@
+
+      - id: SpaceCash100
+        prob: 0.90
+        orGroup: Generic3
+      - id: WhoopieCushion
+        prob: 0.025
+        orGroup: Generic3
+      - id: PlasticBanana
+        prob: 0.025
+        maxAmount: 5
+        orGroup: Generic3
+      - id: PillSpaceDrugs
+        prob: 0.025
+        maxAmount: 5
+        orGroup: Generic3
+      - id: StrangePill
+        prob: 0.025
+        maxAmount: 5
+        orGroup: Generic3
+      - id: SpaceCash
+        prob: 0.025
+        orGroup: Generic3  #5x tiny
+
+      - id: SpaceCash100
+        prob: 0.85
+        orGroup: Clothing1
+      #- id: ClothingHeadHatCatEars
+      #  prob: 0.15
+      #  orGroup: Clothing1
+      - id: ClothingHeadHatOutlawHat
+        prob: 0.15
+        orGroup: Clothing1
+      - id: ClothingMaskItalianMoustache
+        prob: 0.15
+        orGroup: Clothing1
+      - id: ClothingHandsGlovesCombat
+        prob: 0.15
+        orGroup: Clothing1
+      - id: ClothingNeckScarfStripedSyndieRed
+        prob: 0.15
+        orGroup: Clothing1
+      - id: SpaceCash
+        prob: 0.15
+        orGroup: Clothing1  #1x small
+
+      - id: SpaceCash100
+        prob: 0.95
+        orGroup: Cannabis
+      - id: Joint
+        prob: 0.25
+        maxAmount: 5
+        orGroup: Cannabis
+      - id: Blunt
+        prob: 0.25
+        maxAmount: 5
+        orGroup: Cannabis
+      - id: SmokingPipeFilledCannabis
+        prob: 0.25
+        maxAmount: 5
+        orGroup: Cannabis
+      - id: GroundCannabis
+        prob: 0.25
+        maxAmount: 15
+        orGroup: Cannabis
+      - id: SpaceCash
+        prob: 0.25
+        orGroup: Cannabis #5x tiny
+
+      - id: SpaceCash100
+        prob: 0.98
+        orGroup: GizmosTrigger
+      - id: TimerTrigger
+        prob: 0.01
+        maxAmount: 2
+        orGroup: GizmosTrigger
+      - id: SignalTrigger
+        prob: 0.01
+        maxAmount: 2
+        orGroup: GizmosTrigger
+      - id: VoiceTrigger
+        prob: 0.01
+        maxAmount: 2
+      - id: ProximitySensor
+        prob: 0.01
+        orGroup: GizmosTrigger
+      - id: SpaceCash
+        prob: 0.01
+        orGroup: GizmosTrigger #2x small
+
+      - id: SpaceCash100
+        prob: 0.95
+        orGroup: Chems
+      - id: UnstableMutagenChemistryBottle
+        prob: 0.05
+        maxAmount: 5
+        orGroup: Chems
+      - id: Left4ZedChemistryBottle
+        prob: 0.05
+        maxAmount: 5
+        orGroup: Chems
+      - id: EZNutrientChemistryBottle
+        prob: 0.05
+        maxAmount: 5
+        orGroup: Chems
+      - id: RobustHarvestChemistryBottle
+        prob: 0.05
+        maxAmount: 5
+        orGroup: Chems
+      - id: EpinephrineChemistryBottle
+        prob: 0.05
+        maxAmount: 5
+        orGroup: Chems
+      - id: EphedrineChemistryBottle
+        prob: 0.05
+        maxAmount: 5
+        orGroup: Chems
+      - id: VestineChemistryVial
+        prob: 0.05
+        orGroup: Chems
+      - id: OmnizineChemistryBottle
```

Comment:
The only purpose for these is antag stuff, I don't think its a good idea to place them around the station where passengers will find them.

Reply:
Oops I didn't mean to include the omni there, Omni is probably fine.